Makefile

```diff
@@ -513,6 +516,13 @@ ci.e2e.build.%:
 
 e2e.build.centos-7: centos7 docker.clean-latest-ami
 
+# Run os-packages-artifacts outside devkit container.
+e2e.build.centos-7-offline:
+	$(MAKE) os-packages-artifacts pip-packages-artifacts
```

Comment:
artifacts created by`os-packages-artifacts` and `pip-pakages-artifacts` are unavailable to devkit container. so running them on the host.
`make os-packages-artifacts` launches another docker container (rpm-getter) which mounts the `artifacts` directory from the absolute path the host. This directory path will be unavailable to devkit container if the "rpm-getter" container is created from within devkit container. 

Another alternative is to create docker volume and share between "devkit" and "rpm-getter" container.